packaging/deb/freerdp-nightly/freerdp-nightly.install

```diff
@@ -1,3 +1,3 @@
-opt/freerdp-nightly/lib/*.so.*
```

Comment:
This was intentional. The library it self is in the freerdp-nigthly package but the .so files (without version), which are only required for compilation, are shipped with the freerdp-nightly-dev. This is the way debian (and also rpm packages) usually do it.